Assignment.Tests/SampleDataTests.cs

```diff
@@ -0,0 +1,120 @@
+        foreach (var row in csvRowsFromSampleData)
+        {
+            var columns = row.Split(',');
+            Assert.AreEqual(expectedColumnCount, columns.Length, $"Incorrect number of columns in CSV row: {row}");
+        }
+
+        // Check if the CSV rows from the file match the ones from SampleData
+        Assert.IsTrue(csvRowsFromFile.SequenceEqual(csvRowsFromSampleData));
+        }
+
+
+    [TestMethod]
+    public void People_PeopleCSV_ReturnsSortedListOfPeople()
+    {
+        // Convert CSV rows into formatted strings representing people's data
+        IEnumerable<string> personsFromCsv = SampleData.CsvRows
+            .Select(line => line.Split(","))
+            .OrderBy(line => (line[5], line[6], line[7]))
+            .Select(line => string.Join(",", line.Skip(1)));
+
+        // Convert people objects into formatted strings representing people's data
+        IEnumerable<string> personsFromObjects = SampleData.People
+            .Select(person =>
+                $"{person.FirstName},{person.LastName},{person.EmailAddress},{person.Address.StreetAddress},{person.Address.City},{person.Address.State},{person.Address.Zip}");
+
+        // Assert that both sequences of formatted strings are equal
+        Assert.IsTrue(personsFromCsv.SequenceEqual(personsFromObjects));
+    }
+
+
+    [TestMethod]
+    public void GetUniqueSortedListOfStatesGivenCsvRows_LINQ_ReturnsDistinctSortedStateList()
+    {
+        // Get the list of states from SampleData method
+        var statesFromSampleData = SampleData.GetUniqueSortedListOfStatesGivenCsvRows();
+
+        // Get the list of states directly from the CSV rows, split them, and select the state
+        var statesFromCsv = SampleData.CsvRows
+            .Select(row => row.Split(',')[6].Trim()) // Extract and trim the state from each CSV row
+            .OrderBy(state => state) // Sort the states
+            .Distinct(); // Get the distinct states
+
+        // Assert that both lists contain the same elements
+        CollectionAssert.AreEqual(statesFromCsv.ToList(), statesFromSampleData.ToList());
+    }
+
+
+
+    [TestMethod]
+    public void FilterByEmailAddress_FilterByEducationEmail_ReturnsPeopleWithGovEmails()
+    {
+
+        Assert.IsTrue(new List<(string, string)> { ("Amelia" , "Toal"), ("Arthur" ,"Myles"), ("Ev", "Challace"), ("Marijn", "McKennan"), ("Priscilla", "Jenyns") }.SequenceEqual(SampleData.FilterByEmailAddress((email) => email.Contains(".gov")).OrderBy(fullName => fullName.FirstName)));
+
+    }
+
+    [TestMethod]
+    public void GetAggregateSortedListOfStatesUsingCsvRows_GivenOrderedList_Equal()
+    {
+        string expectedList = "AL, AZ, CA, DC, FL, GA, IN, KS, LA, MD, MN, MO, MT, NC, NE, NH, NV, NY, OR, PA, SC, TN, TX, UT, VA, WA, WV";
+        string actualList = string.Join(" ", SampleData.GetAggregateSortedListOfStatesUsingCsvRows());
+        Assert.IsTrue(expectedList == actualList);
```

Comment:
```suggestion
        Assert.AreEqual(expectedList, actualList);
```
Use Assert methods over doing it yourself